generator.rb

```diff
@@ -0,0 +1,73 @@
+# Validates if value in word
+def is_vowel?(letter)
+  if letter == "a" || letter == "e" || letter == "i" || letter == "o" || letter == "u"
+    return true
+  else
+    return false
+  end
+end
+
+# Gather user input of first and second part of portmanteau. Verify at least 2 characters long.
+def run_generator
+  puts "What is the first word to combine?: "
+  first_word = gets.chomp.downcase
+  until first_word.length >= 2
+    puts "Error. Please submit a word with at least 2 characters:"
+    first_word = gets.chomp
+  end
+
+  puts "What is the second word to combine?: "
+  second_word = gets.chomp.downcase
+  until second_word.length >= 2
+    puts "Error. Please submit a word with at least 2 characters:"
+    second_word = gets.chomp
+  end
```

Comment:
The logic to check the validity of the input is very similar. This would be a good opportunity to make a helper method!